Dockerfile

```
@@ -1,14 +1,15 @@
-FROM golang:1.24
+FROM golang:1.25.11
 ADD . /usr/src/whereabouts
```

Hadolint (2.15.1)

[error] Use COPY instead of ADD for files and folders

(DL3020)